front/components/vaults/ContentActions.tsx

```diff
@@ -14,10 +14,8 @@ import { MultipleDocumentsUpload } from "@app/components/data_source/MultipleDoc
 import { isFolder, isWebsite } from "@app/lib/data_sources";
 
 export type ContentActionKey =
-  | "EditDocument"
-  | "EditTable"
-  | "CreateDocument"
-  | "CreateTable"
+  | "DocumentUploadOrEdit"
+  | "TableUploadOrEdit"
   | "MultipleDocumentsUpload"
   | "DocumentOrTableDeleteDialog"
   | "TableUploadOrEditModal";
```

Comment:
`nit`: not sure if this is still being used? 🤔

Reply:
good catch 👍